.github/workflows/docker-validate.yml

```
@@ -0,0 +1,192 @@
+          # Display the .env file for debugging
+          echo "Contents of .env file:"
+          cat .env
+
+      - name: Validate docker-compose.yml
+        run: |
+          docker compose config -q
+          if [ $? -ne 0 ]; then
+            echo "docker-compose.yml is not valid"
+            exit 1
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)